file_paths.md

```diff
@@ -0,0 +1,106 @@
+File Paths
+==========
+
+In Puppet 4.0, file paths and related settings will be changing as follows.
```

Comment:
"This table specifies the file paths in an installation of Puppet and the corresponding setting. The 3x. column shows the difference of this version of the specification vs. the actual 3x implementation."

(It is really not up to the specification to state which version of an implementation that complies with the spec).